person-service/src/main/kotlin/com/thehuginn/controller/AddressResource.kt

```diff
@@ -0,0 +1,19 @@
+package com.thehuginn.controller
+
+import com.thehuginn.controller.dto.response.mapper.AddressResponseMapper
+import com.thehuginn.service.AddressService
+import jakarta.ws.rs.GET
+import jakarta.ws.rs.Path
+import java.util.UUID
+
+@Path("/address")
+class AddressResource(
+    private val addressService: AddressService,
+    private val addressResponseMapper: AddressResponseMapper
+) {
+
+    @GET
+    @Path("/{id}")
```

Comment:
zmenit EP (endpoint), nech berie jeden queryParam namiesto pathParam a to `userId`. Nasledne vyhladat aj v service podla `userId`

Reply:
Prerobím na akceptovanie Query paramu namiesto Path, ale radšej spravím separátny endpoint podľa userId, keďže ten bude vraciať List<AddressResponse>